### PR TITLE
chore: drop dependency on @google-cloud/common

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -28,9 +28,9 @@ const is = require('is');
 const PKG = require('../package.json');
 const v1 = require('./v1');
 
-const Snapshot = require('./snapshot.js');
-const Subscription = require('./subscription.js');
-const Topic = require('./topic.js');
+const Snapshot = require('./snapshot');
```
Comment (ofrobots) [inline src/index.js]: <insert grump about unrelated changes in a commit here>

Comment (JustinBeckwith) [reply src/index.js]: apologies ❤️
